views/forms/UserProfileForm/UserProfileForm.test.jsx

```diff
@@ -51,30 +51,26 @@ describe('User Profile Form', () => {
           display_name: 'My Full Name',
           mail: 'myEmail@example.test',
           title: 'My Title',
-          department: 'My Department',
-          company: 'My Company',
-          icon: expect.stringContaining('data:image/png;base64'),
+          icon: '',
+          iconFileName: '',
```

Comment:
we shouldn't have icon and icon file name in the form data, we only need the icon file.

Reply:
[0730f7e](https://github.com/AMP-SCZ/dpdash/pull/478/commits/0730f7e99359b47d86b078fb3e0ad15f74bb8fcb)